src/docs/dynamic-sampling/fidelity-and-biases.mdx

```diff
@@ -0,0 +1,108 @@
+
+Sentry has already defined a set of biases that are available to all customers. These biases have different goals, but they can be combined to express more complex semantics.
+
+Some of the biases defined by Sentry can be enabled or disabled in the UI, more specifically under **Project Settings -> Performance**, while others are enabled by default and cannot be disabled.
+
+An example of how the UI looks is shown in the following screenshot (the content of this screenshot is subject to change):
+
+![Biases in the UI](/images/biasesUI.png)
+
+### Deprioritize Health Checks
+
+This bias is used to de-prioritize transactions that are classified as health checks. The goal is to reduce the amount of data retained for health checks, since they are not very useful for debugging.
+
+In order to mark a transaction as a health check, we leverage a list of known health check endpoints, which is maintained by Sentry and updated regularly.
+
+```python
+HEALTH_CHECK_GLOBS = [
+    "*healthcheck*",
+    "*healthy*",
+    "*live*",
+    "*ready*",
+    "*heartbeat*",
+    "*/health",
+    "*/healthz",
+]
+```
+
+The list of health check endpoints is available [here](https://github.com/getsentry/sentry/blob/4cb0d863de1ef8e3440153cb440eaca8025dee0d/src/sentry/dynamic_sampling/rules/biases/ignore_health_checks_bias.py#L14).
+
+### Prioritize Dev Environments
+
+This bias is used to prioritize transactions coming from a development environment in order to increase the amount of data retained for such environments, since they are more likely to be useful for debugging.
+
+To mark a transaction as a development environment, we leverage a list of known development environments, which is maintained and updated regularly by Sentry.
+
+```python
+ENVIRONMENT_GLOBS = [
+    "*debug*",
+    "*dev*",
+    "*local*",
+    "*qa*",
+    "*test*",
+]
+```
+
+The list of development environments is available [here](https://github.com/getsentry/sentry/blob/4cb0d863de1ef8e3440153cb440eaca8025dee0d/src/sentry/dynamic_sampling/rules/biases/boost_environments_bias.py#L7).
+
+### Prioritize New Releases
+
+This bias is used to prioritize transactions that are coming from a new release. The goal is to increase the sample rate in the time window that occurs between the creation of a release and its adoption by users.
+
+Since the adoption of a release is not constant, we created a system of *decaying* rules which can interpolate between two sample rates in a given time window with a given function (e.g. `linear`).
+
+The latest release bias uses a decaying rule to interpolate between a starting sample rate and an ending sample rate over a time window that is statically defined for each platform. For example, Android has a bigger time window than Javascript because Android apps take more time to get adopted by users.
+
+### Prioritize Low Volume Transactions
```

Comment:
I thought we were calling it transaction rebalancing but for some reason, it was agreed to call it low-volume transactions. I would keep it as is for now and only disambiguate in case we see the need in the future because I would also like to refactor it in code.